### ISSUE TITLE
[Accessibility] Links in a cell doesn't receive tab focus

### ISSUE THREAD

**Issue by abhipanda** [ag-grid/ag-grid#2311]
```
[ x] bug report
```

**Current behavior**

I have a cellRenderer with a simple link - tabbing into the grid doesn't get the focus on the link instead goes from cell to cell. If I use "suppressCellSelection" nothing gets focus.

This is what I see on inspecting:

```html
<div tabindex="-1" role="gridcell" comp-id="24" col-id="recordNumber" class="ag-cell ag-cell-not-inline-editing ag-cell-no-focus ag-cell-value" style="width: 150px; left: 30px;">
<span data-v-6a3684aa="" class="cellLink">
<a data-v-6a3684aa="" href="/test/234">234</a>
</span>
</div>
```

**Expected behavior**
The grid should behave as a normal grid when there is no edit functionality. It should honor regular grid tabindex.

**Comment by makinggoodsoftware** [comment]
Hi,

Thank you for raising this issue for us.

As I am sure you are aware, we offer two products:

ag-Grid - this is entirely free to use in line with our MIT license. (issues reported via GitHub)
ag-Grid Enterprise - our commercial version which funds the development of both versions. (issues reported via zendesk)

If you are a customer, you should be using ag-Grid Enterprise in which case you should have access to our customer support system (via zendesk), and that is the only channel where we accept feature requests.

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.
